README.md

```diff
@@ -1,2 +1,5 @@
 # hello-world-2.0
 Second round in tutorial
+
+
+Hello. I always like doing these random little blurbs about myself becaues I get a chance to reflect a bit about who I think I am, as well as spend a little time thinging about things that make me happy, like my dog, Pepper. Much of the time I end up writing about her because she is so cute and I love her so much. It is just so easy to do. Anyway, I think that this is sufficient text. Tootles!
```

Comment:
Okay, this is what I was specifically hoping for. To be able to look at and comment on very specific pieces of code. I think that this will be one of the more valuable parts for our project and for learning and asking questions as we work.